cps/src/main/js/workflow-editor.js

```diff
@@ -48,7 +48,46 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
                 });
 
                 editor.on('blur', function() {
-                    textarea.trigger('change');
+                    editor.session.clearAnnotations();
+                    var url = textarea.attr("checkUrl") + 'Json';
+
+                    $.ajax({
+                        url: url,
+                        data: {
+                            value: editor.getValue(),
+                            sandbox: $("input[name='_.sandbox']").is(':checked')
+                        },
+                        method: textarea.attr("checkMethod") || "get",
```

Comment:
May as well hardcode `POST`. You do not want `GET` as it will not handle large scripts.